pom.xml

```
@@ -19,7 +26,42 @@
     <quarkus.platform.group-id>io.quarkus</quarkus.platform.group-id>
     <quarkus.platform.version>1.9.0.Final</quarkus.platform.version>
     <surefire-plugin.version>3.0.0-M5</surefire-plugin.version>
+
+    <windup.web.scm.connection>scm:git:https://github.com/windup/windup-operator.git</windup.web.scm.connection>
+    <windup.web.developer.connection>scm:git:git@github.com:windup/windup-operator.git</windup.web.developer.connection>
+    <windup.web.scm.url>http://github.com/windup/windup-operator</windup.web.scm.url>
   </properties>
+
+  <scm>
+      <tag>master</tag>
+      <connection>${windup.web.scm.connection}</connection>
+      <developerConnection>${windup.web.developer.connection}</developerConnection>
+      <url>${windup.web.scm.url}</url>
+  </scm>
+
+  <issueManagement>
+      <system>JBoss Jira</system>
+      <url>https://issues.jboss.org/browse/WINDUP</url>
```

I think it's the "legacy" URL that works thanks to a redirect.
In this case we're adding it for the first time so better use the right URL
```suggestion
      <url>https://issues.redhat.com/browse/WINDUP</url>
```

---

:+1: